sources/packages/web/src/components/institutions/offerings/OfferingForm.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <formio
```

Comment:
Is this component intended to be shared between Institution and Ministry? If yes if should not be on Institituins folders, right?

Reply:
I was not able to see this file moved. Is the PR updated with this change?